### PR TITLE
update sample config to solve bug

### PR DESCRIPTION
**Pull request by gregtampa**
Short Description: 

Fixes:
- pokemon dont get caught and spam
- 
- 

buggy:
"catch_above_cp": 0, "catch_above_iv": 0.8, "logic": "and"
replaced:
"always_catch": true

### CODE REVIEW

File: configs/config.json.pokemons.example [hunk 1]
```diff
@@ -57,18 +57,18 @@
...
 
       "// Magikarp evolves to Gyarados": {},
-      "Magikarp": { "catch_above_cp": 91, "catch_above_iv": 0.9, "logic": "and" },
+      "Magikarp": { "always_catch": true },
       "// Exeggcute evolves to Exeggutor": {},
-      "Exeggcute": { "catch_above_cp": 384, "catch_above_iv": 0.8, "logic": "and" },
+      "Exeggcute": { "always_catch": true },
       "// Eevee evolves to many versions, like Vaporeon, Flareon": {},
-      "Eevee": { "catch_above_cp": 376, "catch_above_iv": 0.8, "logic": "and"},
+      "Eevee": { "always_catch": true },
```
Comment (Zgrkaralar) [inline configs/config.json.pokemons.example]: Eevee :)

**Comment by mrseeker** [comment]
I am testing it, and it seems to catch magikarps... which is a good sign, since it was bothering me like hell.

```
[07:48:02] A Wild Magikarp appeared! [CP 113] [Potential 0.58]
[07:48:02] IV [Stamina/Attack/Defense] = [5/13/8]
[07:48:06] Catch Rate with normal Pokeball is low (49.40%). Throwing Razz Berry... (10 left!)
[07:48:06] Catch Rate with normal Pokeball has increased to 74.11%
[07:48:06] Using Pokeball (chance: 74.11%)... (46 left!)
[07:48:07] Captured Magikarp! [CP 113] [Potential 0.58] [5/13/8]
[07:48:07] Failed to evolve Magikarp!
```

**Comment by douglascamata** [comment]
👍

**Comment by tstumm** [comment]
👍

**Comment by Zgrkaralar** [comment]
👍

**Pull request merged**